_extras/checkout.md

```diff
@@ -70,10 +70,9 @@ All Carpentries curricula (including this one) are hosted on GitHub. Learning to
 including the ability to contribute to other open-source projects! However, we understand that there are many
 reasons why trainees may wish to avoid engaging on GitHub. That's ok!
 
-For this checkout task, you may email your contribution to [instructor.training@carpentries.org](mailto:instructor.training@carpentries.org).
-Be sure to include a link to the lesson or site that you are addressing in your contribution. A Carpentries Core Team member will
-add your contribution to the relevant repository on your behalf (e.g. by creating an issue), and will send you a link so that
-you may view any responses by the maintainers.
+
+For this checkout task, please log in to your [AMY profile](https://docs.carpentries.org/topic_folders/for_instructors/current_instructors.html#accessing-and-updating-your-instructor-profile) and select “Training progress” at the top of the screen.  You can now enter in a link to your lesson contribution.  This will be evaluated in 7-10 days.  If you are having trouble logging into AMY, you may contact  [instructor.training@carpentries.org](mailto:instructor.training@carpentries.org) and we will help you log in. If you do not use GitHub, you may submit your lesson contribution via [this form](https://docs.google.com/forms/d/e/1FAIpQLSeMBOj5Rdh8Mgk0ebRbeRyHhGHKbItdft6avWuEzmeg8CgWbA/viewform).  A Carpentries Core Team member will create an issue based on the relevant repository, and will send you a link so that you may view any responses. In either case, you can track your progress by [logging into AMY here](https://amy.carpentries.org).
```

Comment:
```suggestion
For this checkout task, please log in to your [AMY profile](https://docs.carpentries.org/topic_folders/for_instructors/current_instructors.html#accessing-and-updating-your-instructor-profile) and select “Training progress” at the top of the screen.  You can now enter in a link to your lesson contribution.  
This will be evaluated in 7-10 days.  If you are having trouble logging into AMY, you may contact  [instructor.training@carpentries.org](mailto:instructor.training@carpentries.org) and we will help you log in. 
If you do not use GitHub, you may submit your lesson contribution via [this form](https://docs.google.com/forms/d/e/1FAIpQLSeMBOj5Rdh8Mgk0ebRbeRyHhGHKbItdft6avWuEzmeg8CgWbA/viewform).  
A Carpentries Core Team member will create an issue based on the relevant repository, and will send you a link so that you may view any responses. 
In either case, you can track your progress by [logging into AMY here](https://amy.carpentries.org).
```